docker-compose.prod.yaml

```diff
@@ -75,8 +75,10 @@ services:
         condition: service_started
       minio:
         condition: service_started
+    env_file:
+     - .env
```

Comment:
Please remove. This will cause docker to fail if there is no `.env` file, which is already happening